main.py

```diff
@@ -238,6 +237,8 @@ def wait_until_visible(driver, xpath=None, class_name=None, duration=10000, freq
     parser.add_argument("--select-payment", action="store_true")
     parser.add_argument("--purchase", action="store_true")
     parser.add_argument("--num-retries", type=int, default=1)
+    parser.add_argument("--dont-quit", action="store_true")
+    parser.add_argument("--shoe-type", default="M")
```

Comment:
I'd prolly just add `choices=("M", "F", "W", "Y")` here and be done with it. I wouldn't bother accepting all those variations you have below.

Reply:
Sure no problem! I just figured from a user experience standpoint (user's who aren't technically inclined or forget what to input as a parameter) to just cover all the bases for them. I will remove. 

Also, just realized, some shoes only have the shoe size, so I'll have to somehow check before searching for the element and set the string accordingly. I'll work on it and include in a future update.

Reply:
If you add `choices=("M", "F", "W", "Y")` as a param for `add_argument`, the user will get an error if they use something outside of those accepted values.